### DISCUSSION TITLE
Help contributing

### DISCUSSION THREAD

**Discussion by xa31er** [Infinite-Chess/infinitechess.org#88] [Q&A] [answered]
Hello, I would like to help with this project, but I don't really know what is needed. Having both GitHub issues and Discord messages is a bit confusing for me.

Thanks for all the help!

**Comment by tsevasa** [comment, marked as answer]
The board editor and the porting of the piece sprites from pngs to svgs are two upstart projects discussed on the discord server where not a lot has happened yet. Other than that, feel free to browse the open issues on github, they all still need work!

**Reply by xa31er** [reply]
Thank you, I will get right on it!
